tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -200,7 +200,7 @@ public int getClientMaxPendingRequests() {
         private EventLoopGroup bossGroup = new NioEventLoopGroup(1);
         private EventLoopGroup childGroup = new NioEventLoopGroup();
         private LogLevel logLevel = LogLevel.INFO;
-        private int initTimeout = 2000;
+        private long initTimeout = -1;
```

Comment:
If timeout == -1, it will use the req timeout:
https://github.com/uber/tchannel-java/pull/108/files#diff-6f54a004122ffb4db1cbf0ed46cde280R269